module/bdev/raid/bdev_raid.c

```diff
@@ -1644,13 +1757,14 @@ raid_bdev_channel_remove_base_bdev(struct spdk_io_channel_iter *i)
 }
 
 static void
-raid_bdev_remove_base_bdev_write_sb_cb(bool success, struct raid_bdev *raid_bdev)
+raid_bdev_remove_base_bdev_write_sb_cb(bool success, struct raid_bdev *raid_bdev, void *ctx)
```

Comment:
Nothing in the build fusses about unused parameters?

Reply:
Build rise a warning when you have unused variable declared in the function body, but for it all is fine if you have unused function parameters